public/js/SettingIconLastBuyPriceRemoveThreshold.js

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable react/jsx-no-undef */
+/* eslint-disable no-undef */
+class SettingIconLastBuyPriceRemoveThreshold extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      lastBuyPriceRemoveThresholds: {}
+    };
+
+    this.handleInputChange = this.handleInputChange.bind(this);
+  }
+
+  componentDidUpdate(nextProps) {
+    // Only update configuration, when the modal is closed and different.
+    if (
+        _.isEmpty(nextProps.lastBuyPriceRemoveThresholds) === false &&
+        _.isEqual(nextProps.lastBuyPriceRemoveThresholds, this.state.lastBuyPriceRemoveThresholds) ===
+        false
+    ) {
+        const { lastBuyPriceRemoveThresholds } = nextProps;
+        console.log('lastBuyPriceRemoveThreshold has changed', { lastBuyPriceRemoveThresholds });
+      this.setState({
+        lastBuyPriceRemoveThresholds
+      });
+    }
+  }
+
+  handleInputChange(event) {
+    const target = event.target;
+    const value =
+      target.type === 'checkbox'
+        ? target.checked
+        : target.type === 'number'
+        ? +target.value
+        : target.value;
+    const stateKey = target.getAttribute('data-state-key');
+
+      const { lastBuyPriceRemoveThresholds } = this.state;
+
+    console.log(
+      '_.set(lastBuyPriceRemoveThresholds, stateKey, value) => ',
+        _.set(lastBuyPriceRemoveThresholds, stateKey, value)
+    );
+
+      const newLastBuyPriceRemoveThreshold = _.set(lastBuyPriceRemoveThresholds, stateKey, value);
+    this.setState({
+        lastBuyPriceRemoveThresholds: newLastBuyPriceRemoveThreshold
+    });
+
+      this.props.handleLastBuyPriceRemoveThresholdChange(lastBuyPriceRemoveThresholds);
+  }
+
+  render() {
+    const { quoteAssets } = this.props;
+    const { lastBuyPriceRemoveThresholds } = this.state;
+
+    if (_.isEmpty(lastBuyPriceRemoveThresholds)) {
+      return '';
+    }
+
+    return quoteAssets.map((quoteAsset, index) => {
+      return (
+        <div
+          key={quoteAsset + '-' + index}
+          className='coin-info-last-buy-threshold-wrapper'>
+          <Form.Group
+            controlId={'field-min-last-buy-limit-percentage-' + quoteAsset}
+            className='mb-2'>
+            <Form.Label className='mb-0'>
+              Last Buy Threshold for {quoteAsset}{' '}
```

Comment:
I just checked UI and found it's a bit confusing.

It should say `Last Buy Price Remove Threshold for {quoteAsset}`.
But I also found the configuration UI is very congested as well. 
Should find a better way to layout the fields.

![image](https://user-images.githubusercontent.com/5715919/120193819-306c8780-c260-11eb-9c1a-f27298921550.png)

Reply:
![image](https://user-images.githubusercontent.com/5715919/120194882-855ccd80-c261-11eb-9eb3-b7bb383da964.png)


Actually, this label/information message seems alright?
You had two different message global/symbol settings UI.

Reply:
Fixed. It was a test message that I forgot to remove it. Now it is right.